content/docs/pipelines/upgrade.md

```diff
@@ -4,19 +4,42 @@ description = "How to upgrade or reinstall your Kubeflow Pipelines deployment"
 weight = 50
 +++
 
-Starting from Kubeflow version 0.5, Kubeflow Pipelines persists the
-pipeline data in a permanent storage volume. Kubeflow Pipelines therefore
+Starting from Kubeflow v0.5, Kubeflow Pipelines persists the
+pipeline data in permanent storage volumes. Kubeflow Pipelines therefore
 supports the following capabilities:
 
 * **Reinstall:** You can delete a cluster and create a new cluster, specifying
-  the storage to retrieve the original data in the new cluster.
+  the existing storage volumes to retrieve the original data in the new cluster.
 
-Note that upgrade isn't currently supported for Kubeflow. Check [this issue](https://github.com/kubeflow/kubeflow/issues/3727)
-for progress.
+* **Upgrade (limited support):**
 
-If you want the latest Kubeflow Pipelines in Google Cloud Platform (GCP), we recommend that you use the [Kubeflow Pipelines standalone deployment](/docs/pipelines/standalone-deployment-gcp/).
```

Comment:
Hi @sarahmaddox, I think we should adjust phrase of this statement to redirect standalone users to corresponding upgrade guide.

Reply:
Done.